cpp/splinepy/py/py_multi_patch.hpp

```diff
@@ -1857,6 +1919,9 @@ inline void add_multi_patch(py::module& m) {
       .def_property_readonly("dim", &PyMultiPatch::Dim)
       .def_property_readonly("name", &PyMultiPatch::Name)
       .def_property_readonly("whatami", &PyMultiPatch::WhatAmI)
+      .def_property_readonly("control_points", &PyMultiPatch::GetControlPoints)
+      .def_property_readonly("control_point_offsets",
```

Comment:
```suggestion
      .def("control_point_offsets",
```
I'd prefer making this a method instead of property, just to give an impression that it is not for free

Reply:
unless we save those

Reply:
I agree, this operation is pretty cheap, so we shouldn't bother with storing it anywhere